db_provision.mjs

```diff
@@ -4,13 +4,19 @@
 import { db } from "@vercel/postgres";
 import * as dotenv from "dotenv";
 dotenv.config({
-  path: ".env.development.local",
+  path: ".env.local",
 });
 
 const client = await db.connect();
 
 // createProductsTable();
-fetchAndInsertProducts("mask");
+await fetchAndInsertProducts("mask");
```

Comment:
I suggest you merge this PR before https://github.com/damdiana/beauty-app/pull/84. Both target the same file, and you will have conflicts. But I think it's easier to merge this one first, and then the other one.